docs/getting-started/install-endpoint.asciidoc

```diff
@@ -0,0 +1,87 @@
+[[install-endpoint]]
+[role="x-pack"]
+= Configure and install Elastic Security
+
+
+WARNING: The early release of {ingest-management} is experimental. We invite you to
+install and test these capabilities **in a test environment**. You might run
+into problems and need to modify your setup to get this feature running. Please
+do not enable or use {ingest-management} in a production environment.
+
+
+Like other Elastic integrations, Elastic Endpoint Security can be integrated into the Elastic Agent through {ingest-guide}/ingest-management-overview.html[Ingest Manager]. Upon configuration, the integration allows the Elastic Agent to monitor for events on your host and send data to the Elastic Security app.
+
+NOTE: Configuring the Endpoint Integration on the Elastic Agent requires that the user have permission to use Ingest Manager in Kibana.
+
+[discrete]
+[[security-before-you-begin]]
+== Before you begin
+
+If you're using the Elastic Agent on macOS Mojave (10.14) or later, ensure that you have enabled <<sensor-full-disk-access,Full Disk Access>>. Lastly, review the Elastic Security system requirements.
```

Comment:
This takes you to the esensor steps not the Elastic Endpoint steps.  Enabling esensor will not work for the elastic-endpoint.

Reply:
Are the steps different? Do you still have to enable full disk access for `elastic-endpoint`?

Reply:
Yes, you still need to enable full disk access for elastic-endpoint.

Reply:
Awesome. I'll update the Full Disk Access page in a separate PR.